src/components/Table/GridTableContext.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { PropsWithChildren, useContext } from "react";
+
+type GridTableContextProps = {
+  inTable: boolean;
+};
+
+export const GridTableContext = React.createContext<GridTableContextProps>({
```

Comment:
I was kinda thinking instead of introducing yet-another-context, we could add `FormContext.numberAlignment = left | right` to the existing (but admittedly not used a lot yet) `FormContext`.

I kinda like that b/c then in theory the form fields would only have to check the form context, instead of maybe eventually ~2-3 various contexts.

So it moves the coupling/decision to the parent to decide the settings and push those down.

Wdyt?

Reply:
I've updated with what we chatted about. Thanks for the feedback and input!